package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Browser compatibility data provided by MDN Web Docs",
   "main": "index.js",
   "types": "index.d.ts",
+  "type": "module",
```

Comment:
This may be a problem without landing #15775 first.

Reply:
This will be alright since we generate a stripped-down package.json using only select properties for the releases!